### PR TITLE
Added Tango for Commitment Orchestrator

### PR DESCRIPTION
**Pull request by joeyouss**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/Commitment-orchestrator/overview.md [hunk 1]
```diff
@@ -110,8 +112,22 @@ After all the set-up steps, you can review and finalise your inputs.
 <DocImage path={require('./static/Step4.png')} width="80%" height="80%" title="Click to view full size image" />
 
 Post set-up, you can view your dashboard with all the information required . You can manipulate the information shown according to the filters such as Instances and Regions and see all the information related to Computer Coverage, Savings, Commitment Utilisation alongwith Log history. This way, the dashboard allows you to easily keep a track of your commitments and make informed decisions.
+
 <DocImage path={require('./static/Dashboard_CO.png')} width="80%" height="80%" title="Click to view full size image" />
 
+#### Interactive walkthrough for the same:
+ <iframe 
+     src="https://app.tango.us/app/embed/845a1973-c350-4d4e-b5cc-51ad3482c0f9" 
+     title="Set up a Git Repository Connector for Harness IaCM"
```
Comment (richardblack-Harness) [inline docs/cloud-cost-management/4-use-ccm-cost-optimization/Commitment-orchestrator/overview.md:121]: Should the title be updated here to match the Tango guide?

**Review by sandyydk — APPROVED** [review]
LGTM

**Comment by sandyydk** [comment]
@joeyouss Tango doesn't show instance exclusion step though we see the button to get to that. is that expected?

File: docs/cloud-cost-management/4-use-ccm-cost-optimization/Commitment-orchestrator/overview.md (hunk 1 above)
Comment (joeyouss) [reply docs/cloud-cost-management/4-use-ccm-cost-optimization/Commitment-orchestrator/overview.md]: corrected

**Comment by joeyouss** [comment]
> @joeyouss Tango doesn't show instance exclusion step though we see the button to get to that. is that expected?

We can add that, let me edit screens

**Pull request merged**